Bugzilla/Util.pm

```diff
@@ -254,6 +256,11 @@ sub i_am_webservice {
            || $usage_mode == USAGE_MODE_REST;
 }
 
+sub is_webserver_group {
+    state $web_server_gid = getgrnam(Bugzilla->localconfig->{webservergroup});
```

Comment:
nit:
Bugzilla->localconfig->{webservergroup} can be '' (though, probably only on windows: the default is the first $EGID)
and the return value of getgrnam() can be undefined.